docs/source/tutorial-intro-chipflow-platform.rst

```diff
@@ -0,0 +1,244 @@
+with:
+
+.. code-block:: bash
+
+    make sim-build
+
+After running this, we will have a simulation binary at ``build/sim/sim_soc``. 
+
+We can't run it just yet, as it needs the software/BIOS too. To build the 
+software we run:
+
+.. code-block:: bash
+
+    make software-build
+
+Now that we have our simulation binary, and a BIOS, we can run it:
+
+.. code-block:: bash
+
+    make sim-run
+
+You should see console output like this:
+
+.. code-block:: bash
+
+ 🐱: nyaa~!
+ SoC type: CA7F100F
+ SoC version: 43D6A2C3
+ Flash ID: CA7CA7FF
+ Entering QSPI mode
+ Zeroing initial RAM...
+ Kernel: 00800000
+ DTB: 00F80000
+ DTB magic: FFFFFFFF
+ about to boop the kernel, ganbatte~!
+
+Run the design on a ULX3S board
+-------------------------------
+
+We can also run our design on an FPGA board, currently only the ULX3S is supported.
+
+First we need to build the design into a bitstream for the board:
+
+.. code-block:: bash
+
+    make board-build
+
+This will write a file ``build/top.bit``. As for the simulation, we need the 
+software/BIOS too. 
+
+If we haven't already, build the bios:
+
+.. code-block:: bash
+
+    make software-build
+
+Now, we load the software/BIOS and design onto board (program its bitstream):
+
+.. code-block:: bash
+
+    make board-load-software-ulx3s
+    make board-load-ulx3s
+
+Your board should now be running. For us to check that it's working, we can 
+connect to it via its serial port:
+
+Connecting to your board on macOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Find the serial port for your board, using :bash:`ls /dev/tty.*` or :bash:`ls /dev/cu.*`. 
+  You should see something like ``/dev/tty.usbserial-K00219`` for your board.
+* Connect to the port via the screen utility, at baud ``112200``, with the command:
+  :bash:`screen /dev/tty.usbserial-K00219 115200`.
+* Now, press the ``PWR`` button on your board, which will restart the design.
+* Within ``screen``, you should now see output like:
+   .. code-block:: bash
+
+     🐱: nyaa~!
+     SoC type: CA7F100F
+     SoC version: B79C1FD7
+     Flash ID: EF401800
+     Entering QSPI mode
+     Zeroing initial RAM...
+     Kernel: 00800000
+     DTB: 00F80000
+     DTB magic: FFFFFFFF
+     about to boop the kernel, ganbatte~!
+
+* To exit screen, use ``CTRL-A``, then ``CTRL-\``.
+
+
+Silicon! 
+--------
+
+When you want to go to silicon for your design, the ChipFlow API gets involved.
+
+First we build an `RTLIL file <https://en.wikipedia.org/wiki/Register-transfer_level>`_, 
+which describes the design as registers and gates.
+
+.. code-block:: bash
+
+    make silicon-rtlil
+
+You should now have an `build/my_design.rtlil`.
+
+Send your RTLIL to the ChipFlow cloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At this point, we'll send the RTLIL along with configuration to the ChipFlow 
+API, which will perform silicon-focused checks on the design, and provide 
+information about its running speed:
```

Comment:
suggestion: `which will perform silicon-focused checks on the design, place and route it for the target ASIC technology, and provide information about its area maximum frequency`